bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -109,15 +109,23 @@ spec:
             source /env/_master
             set +o allexport
           fi
-          
+
           OVN_NODES_ARRAY=({{.OVN_NODES}})
-          MASTER_NODE=$(getent ahostsv4 "${OVN_NODES_ARRAY[0]}" | grep RAW | awk '{print $1}')
-          LOCALHOST=$(getent ahostsv4 "${K8S_NODE}" | grep RAW | awk '{print $1}')
+          OVN_NODE_IPS_ARRAY=({{.OVN_NODE_IPS}})
+          MASTER_NODE_IP="${OVN_NODE_IPS_ARRAY[0]}"
+          NODE_INDEX=0
+          for i in "${!OVN_NODES_ARRAY[@]}"; do
+            if [[ "${OVN_NODES_ARRAY[i]}" == "${K8S_NODE}" ]]; then
+              NODE_INDEX=${i}
+              break
+            fi
+          done
+          THIS_NODE_IP="${OVN_NODE_IPS_ARRAY[${NODE_INDEX}]}"
```

Comment:
You don't need to do this; you can pass in the PodIP via the downward API.

Reply:
(see https://kubernetes.io/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information/#capabilities-of-the-downward-api )